<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePwdForm</class>
 <widget class="QWidget" name="ChangePwdForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Management</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>auth_icon.jpg</normaloff>auth_icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Username:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="4" colspan="2">
        <widget class="QLineEdit" name="oldpwdlineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="4" colspan="2">
        <widget class="QLineEdit" name="userlineEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Old Password:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="4" colspan="2">
        <widget class="QPushButton" name="update_btn">
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: rgb(167, 205, 255);
}</string>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>New Password:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Re-enter New Password:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="4" colspan="2">
        <widget class="QLineEdit" name="newpwdlineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="4" colspan="2">
        <widget class="QLineEdit" name="new2pwdlineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="response_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
